src/Directory.Packages.props

```diff
@@ -28,7 +28,7 @@
 
   <!-- Versions from CommonExcludedAssets.props -->
   <ItemGroup>
-    <PackageVersion Include="Microsoft.Bcl.AsyncInterfaces" Version="1.1.1" />
+    <PackageVersion Include="Microsoft.Bcl.AsyncInterfaces" Version="7.0.0" />
```

Comment:
`System.Text.Json` requires 7.0.0+ version.

Should we really avoid shipping it? Ref #2135 and #2098 

@xiang17, @rajkumar-rangaraj?

Reply:
We can have dependency on it, upgrade the version as needed. The `CommonExcludedAssets`'s purpose is to exclude them from being included in the binaries we release.

To my understanding, we exclude them because the customer who has dependency will bring in the assemblies themselves.

Reply:
I understand the reason of `CommonExcludedAssets` - and it should be valid only for dependencies which are only referenced by MySQL.Data and StackExchange.Redis. If it is transitive dependency also by another library we should ship it.
I think that we should not treat `Microsoft.Bcl.AsyncInterfaces` as this kind of assets. The main reason - we ship the latest version of `System.Text.Json`. If the user will reference older version `Microsoft.Bcl.AsyncInterfaces` it can leads to unexpected scenarios.

Reply:
I think all of the netfx libs that are upgraded to the latest must be shipped. There is a very high chance that customer does not bring the required dependency.

Reply:
Nvm. This lib is referenced only for .NET 4.6.2.
Now I have only some doubts where to keep it. I am adding some comment related to this state.

Reply:
@Kielek good point: the exclusion of some libraries depends on the targeted framework. Ideally, we should do the exclusion according to that. Anyway, related but separate problem.